ingeniamotion/wizard_tests/feedbacks_tests/sincos_encoder_test.py

```diff
@@ -0,0 +1,22 @@
+from typing import TYPE_CHECKING, Optional
+
+if TYPE_CHECKING:
+    from ingeniamotion import MotionController
+from ingeniamotion.enums import SensorType
+from ingeniamotion.wizard_tests.feedbacks_tests.feedback_test import Feedbacks
+
+
+class SinCosEncoderTest(Feedbacks):
+    """SinCos encoder test class."""
+
+    FEEDBACK_POLARITY_REGISTER = "FBK_SINCOS_POLARITY"
+
+    BACKUP_REGISTERS_SINCOS: list[str] = [FEEDBACK_POLARITY_REGISTER]
+
+    SENSOR_TYPE_FEEDBACK_TEST = SensorType.SINCOS
+
+    def __init__(
+        self, mc: "MotionController", servo: str, axis: int, logger_drive_name: Optional[str] = None
+    ) -> None:
+        super().__init__(mc, servo, axis, logger_drive_name)
+        self.backup_registers_names.extend(self.BACKUP_REGISTERS_SINCOS)
```

Comment:
https://github.com/ingeniamc/ingeniamotion/blob/0544f27ea101a6f755e530b9f3bf61469bfd80cf/ingeniamotion/wizard_tests/phase_calibration.py#L87C44-L87C60

The attribute is copying the reference to the list instead of creating a new one. So this must be extending the class list, which is very bad.
Don't know if it's an issue with other tests, but at least for here it should be fixed, creating a copy there, so that you can safely extend